msm/src/ffa/columns.rs

```diff
@@ -29,3 +29,22 @@ impl ColumnIndexer for FFAColumnIndexer {
         }
     }
 }
+
+impl FFAColumnIndexer {
```

Comment:
Do you plan to use the same structure than the one in the interpreter?

Reply:
Hm, I'm not exactly sure what the final structure will look like. We can chat tomorrow about it perhaps. 

Some top of the head questions (more for writing down now / discussing later):
1. Both instances of the trait use `Column` to instantiate `Position`. I'm actually not sure if any other thing makes sense, like if it even makes sense to have this parameter.  
2. If we use this parameter, maybe it makes sense to use the Indexer-type thing instead of the raw column. Just to abstract away the "flat" witness representation. Maybe we don't need `column_pos(...)` because the position could be directly expressed by the indexer.
3. These questions generalize it seems: I also want to think a bit about the "very generic interpreter trait" that can be reused across pretty much all the projects (with very minimal methods), as opposed to the "interpreter for concrete set of columns". I think we're leaning towards (2) but then some parts of the abstraction are a signs of (1) which we don't plan to do it seems, and are just rudimentary.

Reply:
On the pragmatic side I'm trying to keep kind of somewhat close to the interpreter in serialization, but thinking if some alternative way could be better, so that we can talk about both / unify them later. If an alternative is not written down it's hard to talk about it :)